vyper/codegen/types/types.py

```diff
@@ -20,7 +20,18 @@
     ABIType,
 )
 from vyper.exceptions import ArgumentException, CompilerPanic, InvalidType
-from vyper.utils import BASE_TYPES, ceil32
+from vyper.utils import ceil32
+
+# Available base types
+UNSIGNED_INTEGER_TYPES = {"uint8", "uint256"}
```

Comment:
The idea here is to add more types in another PR?

Reply:
yes, and also have it available to other modules (e.g. the test suite) which may want to do something applied to "any unsigned integer type"